interview-process/interview-introduction.md

```diff
@@ -0,0 +1,101 @@
+# The Interview Process
+
+The tech hiring process can be stressful, but doing well as a job applicant is a _learned skill_.  You _will likely_ feel awkward and uncomfortable in the beginning, but there is a key secret to remember.
+
+![don't panic](/assets/interview-process_interview-introduction_dont-panic.jpg)
+
+The Ada curriculum and community provide a strong support network to help you succeed in the interview process.
+
+## Goals
+
+- Describe the most common steps in the application process.
+- Understand hiring terminology such as:
+  - Whiteboard Interview
+  - Coding Challenge
+  - Phone Screening
+
+## Vocabulary and Synonyms
+
+| Vocab	|  Definition	| Synonym | How to Use in a Sentence |
+|---	|---	|---	|---	|
+|  Whiteboard Interview  	|  A problem solving exercise where a job applicant is asked to solve a programming or design problem on a whiteboard. 	|    Technical Interview	|   I have a whiteboard interview next week, so I need a buddy to practice with. 	|
+| Phone Screen | An initial telephone or internet chat with a job candidate to ensure the candidate's suitability for an in-peron interview.  This could involve a hiring manager, but is more likely to involve a recruiter. | &nbsp; | I have a phone screening with a recruiter at Dynamic Widgets in 3 hours. | 
+| Coding Challenge | A "homework" assignment given to job candidates to assess their programming skills and practices such as regularly committing and writing tests. |  | I'm trying to write tests for my Dynamic Widgets coding challenge assignment. 
+
+## The Typical Tech Hiring Process
+
+When applying for a junior developer position in the United States, applicants typically follow these steps.
+
+1. Applying for a position
+1. Phone or video chat screening
+1. In-person or video interview
+1. Offer & Negotiation
+1. Acceptance
+
+These steps are typical in the process, but the process can vary greatly.  Companies may not engage in the phone screening or might require additional interviews, or provide homework in the form of a coding challenge.
+
+### Application
+
+A junior developer can apply for a position in many ways, this can include an online application form on the company website or a job-listing site like [Indeed](https://www.indeed.com/q-Apply-jobs.html).  Sometimes it can also include talking to a recruiter, a colleague at the company, or a hiring manager at a local meet-up.  Universities like the University of Washington hold job-fairs.  There are a lot of ways to get in the door as a junior developer, but the most effective strategies typically involve networking.  
```

Comment:
```suggestion
A junior developer can apply for a position in many ways. This can include an online application form on the company website or a job-listing site, like [Indeed](https://www.indeed.com/q-Apply-jobs.html).  Sometimes it can also include talking to a recruiter, a colleague at the company, or a hiring manager at a local meet-up.  Universities like the University of Washington hold job-fairs.  

There are a lot of ways to get in the door as a junior developer, but the most effective strategies typically involve networking.  
```